docs/apis-tools/spring-zeebe-sdk/getting-started.md

```diff
@@ -14,13 +14,27 @@ This project allows you to leverage Zeebe APIs ([gRPC](docs/apis-tools/zeebe-api
 
 ## Add the Spring Zeebe SDK to your project
 
-Add the following Maven dependency to your Spring Boot Starter project:
+Add the following repository and Maven dependency to your Spring Boot Starter project:
+
+```xml
+<repository>
+    <releases>
+        <enabled>true</enabled>
+    </releases>
+    <snapshots>
+        <enabled>false</enabled>
+    </snapshots>
+    <id>identity</id>
+    <name>Camunda Identity</name>
+    <url>https://artifacts.camunda.com/artifactory/camunda-identity/</url>
+</repository>
```

Comment:
🔧 for a better UX let's also add the `<repositories>` wrapper here, so someone could just c&p the snippet into their pom.
```suggestion
<repositories>
    <repository>
        <releases>
            <enabled>true</enabled>
        </releases>
        <snapshots>
            <enabled>false</enabled>
        </snapshots>
        <id>identity</id>
       <name>Camunda Identity</name>
        <url>https://artifacts.camunda.com/artifactory/camunda-identity/</url>
    </repository>
</repositories>
```